wsic/utils.py

```diff
@@ -13,46 +13,31 @@ def downsample_shape(
     downsample: int,
     rounding_func: Callable[[Number], int] = floor,
 ) -> Tuple[int, ...]:
-    r"""Calculate the shape of an array after downsampling by a factor.
-
+    """Calculate the shape of an array after downsampling by a factor.
```

Comment:
Why have you changed the string type here? It will render incorrectly in the documentation if it is not a raw string due to the Sphinx math environment backslashes. This must be a Python raw string or have the backslashes escaped.